recaf-ui/src/main/java/me/coley/recaf/ui/control/code/bytecode/AssemblerArea.java

```diff
@@ -43,467 +49,637 @@
 import org.objectweb.asm.tree.MethodNode;
 import org.slf4j.Logger;
 
+import java.lang.invoke.MethodHandle;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 
 import static me.coley.recaf.ui.control.code.ProblemOrigin.BYTECODE_PARSING;
 import static me.coley.recaf.ui.control.code.ProblemOrigin.BYTECODE_VALIDATION;
+import static me.coley.recaf.ui.util.Menus.action;
+import static me.darknet.assembler.parser.Group.GroupType;
 
 /**
  * Text editing portion of the assembler UI.
  *
  * @author Matt Coley
  */
 public class AssemblerArea extends SyntaxArea implements MemberEditor,
-		AstValidationListener, BytecodeValidationListener, ParserFailureListener, BytecodeFailureListener {
```

Comment:
Your auto-format is set to use spaces, where the rest of Recaf's source is formatted with tabs. The diff here doesn't need to be this large.

Reply:
yes i have noticed